improver_tests/utilities/test_spatial.py

```diff
@@ -199,10 +199,47 @@ def test_axis_keyword(self):
         self.assertAlmostEqual(result, 2 * self.spacing)
 
     def test_lat_lon_equal_spacing(self):
-        """Test outputs with lat-lon grid in degrees"""
+        """Test grid spacing outputs with lat-lon grid in degrees"""
         result = calculate_grid_spacing(self.lat_lon_cube, "degrees")
         self.assertAlmostEqual(result, 10.0)
 
+    def test_lat_lon_equal_spacing_with_tolerance(self):
+        """Test grid spacing outputs with lat-lon grid with tolerance"""
+        self.lat_lon_cube.coord("longitude").points = [
+            -19.99999,
+            -10.0,
+            0.0,
+            10.0,
+            20.00001,
+        ]
```

Comment:
```suggestion
        lon_points = self.lat_lon_cube.coord("longitude").points.copy()
        lon_points[0] += 1e-5
        lon_points[-1] += 1e-5
        self.lat_lon_cube.coord("longitude").points = lon_points
```

Reply:
I also wanted to do your way at the start.  However, I do not know the value of coordinate in the cube. I have to check it to make sure that 1.e-5 works for rtol. The current way is direct, and the reader immediately sees what we are testing and what the result should be.

Reply:
The coordinate points are already implicit here because the expected grid spacing is specified. As you aren't specifying `rtol` in the unit-test, it isn't clear anyway.

Reply:
If you really want to keep the hard-coded definitions, can we have them once, in the setUp function perhaps, rather than three times?